src/ds/d400/d400-private.cpp

```diff
@@ -318,30 +326,30 @@ namespace librealsense
             };
 
             auto table = check_calib< ds::d400_rgb_calibration_table >( raw_data );
-            auto raw_res = width_height_to_ds_rect_resolutions( 1280, 800 );
             auto output_res = width_height_to_ds_rect_resolutions( width, height );
             auto calibration_res = width_height_to_ds_rect_resolutions( table->calib_width, table->calib_height );
 
             float3x3 k = table->intrinsic;
             if( output_res == res_1280_720 )
-                k_to_pixels( k, output_res );
-            else if( output_res == res_640_480 )
+                normalized_k_to_pixels( k, output_res );
+            else if( output_res == res_640_480 ) // 640x480 is 4:3 not 16:9 like other available resolutions. The ISP handling is different.
             {
+                auto raw_res = width_height_to_ds_rect_resolutions( 1280, 800 );
                 // Extrapolate K to raw resolution
                 float scale_y = resolutions_list[calibration_res].y / static_cast< float >( resolutions_list[raw_res].y );
                 k( 1, 1 ) = k( 1, 1 ) * scale_y;  // fy
                 k( 2, 1 ) = k( 2, 1 ) * scale_y;  // ppy
-                k_to_pixels( k, raw_res );
-                // Handle ISP scaling
+                normalized_k_to_pixels( k, raw_res );
+                // Handle ISP scaling to 770x480
                 auto scale_res = width_height_to_ds_rect_resolutions( 770, 480 );
                 scale_pixel_k( k, raw_res, scale_res );
-                // Handle ISP crop
+                // Handle ISP cropping to 640x480
                 k( 2, 0 ) = k( 2, 0 ) - ( resolutions_list[scale_res].x - resolutions_list[output_res].x ) / 2;  // ppx
                 k( 2, 1 ) = k( 2, 1 ) - ( resolutions_list[scale_res].y - resolutions_list[output_res].y ) / 2;  // ppy
             }
             else
             {
-                k_to_pixels( k, calibration_res );
+                normalized_k_to_pixels( k, calibration_res );
```

Comment:
What is the difference between `normalized_k_to_pixels` and `k_to_pixels`?

Reply:
No difference, just renaming based on Remi's comment.
The code always assumes a "normalized k" input and changes it to pixel units based on the requested resolution.